starter-code/src/clue.js

```diff
@@ -84,11 +84,183 @@ name: Guest House
+    firstName:"Eleanor",
+    lastName:"Peacock",
+    color:"blue",
+    description:"She is from a wealthy family and uses her status and money to earn popularity",
+    age: 36,
+    image:"https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation:"Socialité",
+} 
+var mrMustard = {
+    firstName:"Jack",
+    lastName:"Mustard",
+    color:"yellow",
+    description:"He is a former football player who tries to get by on his former glory",
+    age: 62,
+    image:"https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation:"Retired Football player",
+} 
+
+//Room objects
+
+var diningRoom={
+    name:"diningRoom",
+}
+
+var conservatory={
+    name:"conservatory",
+};
+var kitchen={
+    name:"kitchen",
+};
+var study={
+    name:"study",
+};
+var library={
+    name:"library",
+};
+var billiardRoom={
+    name:"billiardRoom",
+};
+var lounge={
+    name:"lounge",
+};
+var ballRoom={
+    name:"ballRoom",
+};
+var hall={
+    name:"hall",
+};
+var spa={
+    name:"spa",
+}
+var livingRoom={
+    name:"livingRoom",
+};
+var observatory={
+    name:"observatory",
+};
+var theater={
+    name:"theater",
+};
+var guestHouse={
+    name:"guestHouse",
+}
+var patio={
+    name:"patio",
+}
+
+//Weapon objects
+var rope={
+    name: "rope",
+    weight: 10,
+} 
+var knife={
+    name: "knife",
+    weight: 8,
+}
+var candleStick={
+    name: "candleStick",
+    weight: 2,
+}
+var dumbbell={
+    name: "dumbbell",
+    weight: 30,
+}
+var poison={
+    name: "poison",
+    weight: 2,
+}
+var axe={
+    name: "axe",
+    weight:15,
+}
+var bat={
+    name: "bat",
+    weight: 13,
+}
+var trophy={
+    name: "trophy",
+    weight: 25,
+}
+var pistol={
+    name: "pistol",
+    weight: 20,
+}
+
 // Characters Collection
 var charactersArray = [];
+charactersArray.push(mrGreen, drOrchid, profPlum, mrMustard, missPeacock, missScarlet);
+//var tempArrChar=[mrGreen, drOrchid, profPlum, mrMustard, missPeacock, missScarlet];
+//for(var i=0; i<tempArrChar; i++){
+
 
 // Rooms' Collection
 var roomsArray = [];
-
+roomsArray.push(diningRoom, conservatory, kitchen, study, library, billiardRoom, lounge, ballRoom, hall, spa, livingRoom, observatory, theater, guestHouse, patio);
 // Weapons Collection
 var weaponsArray = [];
+weaponsArray.push(rope, knife, candleStick, dumbbell, poison, axe, bat, trophy, pistol);
+
+function randomSelector(array){
+    return array[Math.floor(Math.random()*array.length)]  
+    }
+    
+
+function pickMistery(){
+    var misteryEnvelope=[];  
+    misteryEnvelope[0]=randomSelector(charactersArray);
+    misteryEnvelope[1]=randomSelector(weaponsArray);
+    misteryEnvelope[2]=randomSelector(roomsArray);
+        return misteryEnvelope;
+    }
+var envelope= pickMistery();
+
+function revealMistery(envelope){
+    return "" + envelope[0].firstName + " " + envelope[0].lastName + " " + "killed Mr.Boddy using the" + " " + envelope[1].name + " " + "in the" + " " + envelope[2].name + "!!!";
```

Comment:
Save quotes :)
```
    return "" + envelope[0].firstName + " " + envelope[0].lastName + " killed Mr.Boddy using the " + envelope[1].name + " in the " + envelope[2].name + "!!!";

```
instead of 

```
    return "" + envelope[0].firstName + " " + envelope[0].lastName + " " + "killed Mr.Boddy using the" + " " + envelope[1].name + " " + "in the" + " " + envelope[2].name + "!!!";
```